package/Dockerfile

```diff
@@ -4,4 +4,5 @@ ENV KUBECONFIG /root/.kube/config
 RUN microdnf update -y && \
     rm -rf /var/cache/yum
 COPY bin/eks-operator /usr/bin/
+USER 1001
```

Comment:
This can be removed. A more extensive change is coming for the Dockerfiles.

Reply:
I think I accidentally based off a PR of yours or something as it had you co-authored and I don't remember adding anything like that, rebased.